SH4RE/SH4RE/Login/CreateAccount.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  CreateAccount.swift
+//  SH4RE
+//
+//  Created by November on 2023-01-04.
+//
+
+import SwiftUI
+
+struct CreateAccount: View {
+    @Environment(\.showLoginScreen) var showLoginScreen
+    @State private var username: String = ""
+    @State private var password: String = ""
+    @State private var confirmPassword: String = ""
+
+    var body: some View {
+        ZStack {
+            Color(UIColor(customColours["primaryBase"]!))
+                .ignoresSafeArea()
+            Text("Create Account")
+                .foregroundColor(.white)
+                .font(.system(size: 30))
+                .position(x: screenSize.width * 0.5, y: screenSize.height * 0.05)
+
+            Group {
+                VStack {
+                    Spacer()
+                        .frame(height: screenSize.height * 0.05)
+                    
+                    VStack {
+                        Text("Email Address")
+                            .font(.system(size: 18))
+                            .frame(maxWidth: screenSize.width * 0.8, alignment: .leading)
+                        TextField("Your email", text: $username)
```

Comment:
when I ran this on the sim with on-screen keyboard (cmd k), it tried to capitalize the first letter of the email which I guess is the default for text fields. I think for this field, it should be turned off
<img width="301" alt="Screenshot 2023-01-19 at 12 32 20 AM" src="https://user-images.githubusercontent.com/46950469/213363245-78f178ae-2771-43fb-8a35-7d1ad87fb37a.png">